main_example_problem.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+def main():
+
+BILL_AMOUNT = float(input("What is the bill? "))
```

Comment:
@paulghaddad Is there a syntax error in this line? Pylint says there is, but I don't see what it could be? The file and test run properly.